Runtime/Core/ApiGatewayManagement/AmazonGameLiftClientWrapper.cs

```diff
@@ -42,17 +58,181 @@ public async Task<SearchGameSessionsResponse> SearchGameSessions(SearchGameSessi
             return await _amazonGameLiftClient.SearchGameSessionsAsync(request);
         }
 
-        public static IAmazonGameLift Create(string localEndpoint)
+        public async Task<DescribeGameSessionsResponse> DescribeGameSessions(DescribeGameSessionsRequest request)
+        {
+            return await _amazonGameLiftClient.DescribeGameSessionsAsync(request);
+        }
+
+        private IAmazonGameLift Create()
+        {
+            
+            var config = _gameLiftClientSettings.GetGameLiftAnywhereConfiguration();
+            var credentials = SetupCredentials(config.ProfileName);
+            return new AmazonGameLiftClient(credentials.AccessKey, credentials.SecretKey);
+        }
+        #endregion
+        
+        /// <summary>
+        /// Server region is code dedicated to Amazon GameLift SDK and AWS SDK calls made by the game server. All of these calls will be done via UI Elements or on Startup. 
+        /// </summary>
+        #region Server
+
+        private async Task<ListLocationsResponse> ListLocations(ListLocationsRequest request)
+        {
+            return await _amazonGameLiftClient.ListLocationsAsync(request);
+        }
+
+        private async Task<CreateLocationResponse> CreateLocation(CreateLocationRequest request)
+        {
+            return await _amazonGameLiftClient.CreateLocationAsync(request);
+        }
+
+        private async Task<RegisterComputeResponse> RegisterCompute(RegisterComputeRequest request)
         {
-            return new AmazonGameLiftClient(s_dummyAwsAccessKey, s_dummyAwsSecretKey, new AmazonGameLiftConfig
+            return await _amazonGameLiftClient.RegisterComputeAsync(request);
+        }
+
+        private async Task<GetComputeAuthTokenResponse> GetComputeAuthToken(GetComputeAuthTokenRequest request)
+        {
+            return await _amazonGameLiftClient.GetComputeAuthTokenAsync(request);
+        }
+
+        private Task<CreateFleetResponse> CreateFleet(CreateFleetRequest request)
+        {
+            return _amazonGameLiftClient.CreateFleetAsync(request);
+        }
+        
+        private RetriveAwsCredentialsResponse RetrieveAwsCredentials(string profileName)
+        {
+            var request = new RetriveAwsCredentialsRequest { ProfileName = profileName };
+            return _credentialsStore.RetriveAwsCredentials(request);
+        }
+        
+        public async Task UpdateAuthToken()
```

Comment:
I don't see this being used anywhere, is the usage in a future commit? I'd ideally like this to be placed where we are actually requiring the AuhToken. Seems nice to isolate this console from clientSettings. Thoughts?

Reply:
UpdateAuthToken Is used within the Sample Game

Reply:
I will most likely be moving the invokers to our UI Menu as that is where most of them are being used. The invokers in here are temporary until I do. If you would like me to mention that in a comment or a TODO I could do that.

Reply:
yes please, for all the pending tasks please add in a TODO so they don't fall through, and I can approve post that